### ISSUE TITLE
How is the validaiton dataset is used during training?

### ISSUE THREAD

**Issue by meijie0401** [smallcorgi/Faster-RCNN_TF#315]
Hi ,thanks for you tensorflow code. 
I'm training faster rcnn on my own datatset and I already generated train.txt trainval.txt test.txt val.txt.  And now it can be trained on training dataset, but does val.txt is used during training? because I want to see loss on validation dataset.
